app.js

```diff
@@ -1,12 +1,18 @@
 const express = require('express');
 const cors = require('cors');
+const bodyParser = require('body-parser');
```

Comment:
🚫 **[eslint]** <[import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-extraneous-dependencies.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'body-parser' should be listed in the project's dependencies. Run 'npm i -S body-parser' to add it

Reply:
Please do this, but use the yarn command instead.